Towny/src/main/java/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -2935,7 +2935,7 @@ public enum ConfigNodes {
 	
 	ECO_ADVANCED_NPC_UUID_VERSION(
 			"economy.advanced.npc_uuid_version",
-			"2",
+			"4",

```

Comment:
I think we ought to have this set at -1 by default, some people might not understand the default is really 4. Then if another economy plugin is discovered the admin can change it to 2 until we get the eco plugin into the USE_V2_UUID set.